tests/from_msa/test_cluster_sequences.py

```diff
@@ -383,11 +383,11 @@ def test_GivenAllSequencesSmallEditDist_ReturnsNoClustering(self):
             self.assertEqual(expected_clustering, result)
 
     def test_GivenManyVeryDifferentSequences_EachSeqInOwnCluster(self):
-        # all 1024 distinct DNA 5-mers.
-        # We want each in one cluster, but do not want to run clustering 1023 times
-        all_5mers = list(map("".join, product(standard_bases, repeat=5)))
+        # all 256 distinct DNA 5-mers.
+        # We want each in one cluster, but do not want to run clustering 255 times
+        all_5mers = list(map("".join, product(standard_bases, repeat=4)))
         alignment = make_alignment(all_5mers)
-        result = kmeans_cluster_seqs_in_interval([0, 5], alignment, 5)
+        result = kmeans_cluster_seqs_in_interval([0, 4], alignment, 4)
```

Comment:
I'm surprised memory usage on all 5mers was so high, it ran fine on my previous Lenovo laptop. Do you have the log message for this?

(I'm otherwise fine to move down from 5 to 4)

Reply:
When I profiled it locally is peaked at ~150Mb RAM. Which isn't a lot in the context of a laptop, but is quite high for a test. I guess the reason it was only failing when run in a docker build context is maybe the docker daemon on gets a limited amount of RAM for building?

Reply:
no idea- but thanks for digging, is worth knowing that was RAM heavy (not very chuffed by that, hope it isn't an issue on large PRGs)